### PR TITLE
Added register page

### LINKED ISSUE #8: Add Page For Registering User (effort 3)
There is a logic for it in components/Register.js or sth similar to this.

Make it look nice and allow new user to register xd

if smsAllowed then phonenumber textField should be required

### PR DESCRIPTION
**Pull request by GoraJakub**
Also added default theme provider to whole app

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,53 +1,72 @@
...
-import { loadUserInfo, getCookieValue } from './utils';
-import { COOKIE_TOKEN } from './constants';
+import hotelOwnerRoutes from './routes/hotelOwner'
+import userRoutes from './routes/user'
+import { TopSection } from './components'
+import { loadUserInfo, getCookieValue } from './utils'
+import { COOKIE_TOKEN } from './constants'
+import { createMuiTheme, ThemeProvider } from '@material-ui/core/styles'
+
+const theme = createMuiTheme({
```
Comment (pdybowski) [inline src/index.js:16]: is it used somewhere lol

**Review by pdybowski — CHANGES_REQUESTED** [review]
1.
image

2.
moreover Login should be email

3.
add some background to register page. (red means what you must change)
image

File: src/index.js (hunk 1 above)
Comment (GoraJakub) [reply src/index.js:16]: Yes, in theme provider 
image

Comment (pdybowski) [reply src/index.js:16]: Yes, but later :p

**Pull request merged**